services/filemanager/azureBlobStoragemanager.go

```diff
@@ -39,21 +56,25 @@ func (manager *AzureBlobStorageManager) getContainerURL() (azblob.ContainerURL,
 
 	accountName, accountKey := manager.Config.AccountName, manager.Config.AccountKey
 	if len(accountName) == 0 || len(accountKey) == 0 {
-		return azblob.ContainerURL{}, errors.New("Either the AccountName or AccountKey is not correct")
+		return azblob.ContainerURL{}, errors.New("either the AccountName or AccountKey is not correct")
 	}
 
 	// Create a default request pipeline using your storage account name and account key.
 	credential, err := azblob.NewSharedKeyCredential(accountName, accountKey)
 	if err != nil {
 		return azblob.ContainerURL{}, err
 	}
+
 	p := azblob.NewPipeline(credential, azblob.PipelineOptions{})
 
 	// From the Azure portal, get your storage account blob service URL endpoint.
-	URL, _ := url.Parse(
-		fmt.Sprintf("https://%s.blob.core.windows.net/%s", accountName, manager.Config.Container))
+	URL := manager.getBaseURL()
+	url, err := url.Parse(URL)
+	if err != nil {
+		return azblob.ContainerURL{}, err
```

Comment:
Few things here:
1. shadowing the import package `url` is not a good idea.
2. `manager.getBaseURL()` should construct and return a url.URL, it safer this way and you don't need to do Parse and error handle.